gettsim/parameters/grunds_ia.yaml

```
@@ -15,11 +19,12 @@ schonvermögen_grunds_ia:
 
 staatl_rente_anr_frei:
   name:
-    de: Anrechnungsfreie Anteile staatliche Rente wenn mindestens 33 Jahre Grundrentenzeiten
-    en: Public pension shares not subject to transfer withdrawal for subjects whose Grundrentenzeiten exceeds 33 years
+    de: Anrechnungsfreier Anteil der staatlichen Rente für Grundsicherung
```

```suggestion
    de: Anrechnungsfreier Anteil der staatlichen Rente (bei mind. 33 Jahren Grundrentenzeiten)
```

Finde den Verweis auf die 33 Jahre hier wichtig